packages/auth/README.md

```diff
@@ -0,0 +1,27 @@
+# Authorization Server
+
+The authorization server used to issue authorizations for Open Payments actions.
```

Comment:
This could link to:
https://github.com/interledger/open-payments/blob/master/auth-server-open-api-spec.yaml
and/or
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol